### PR TITLE
[WIP] Fix received shares

### PR DESCRIPTION
**Pull request by labkode**

### CODE REVIEW

**Comment by labkode** [comment]
I was looking at https://github.com/cs3org/reva/blob/master/pkg/share/manager/memory/memory.go#L81 to dig deeper

File: internal/http/services/owncloud/ocs/shares.go [hunk 1]
```diff
@@ -651,7 +651,10 @@ func (h *SharesHandler) listSharedWithMe(r *http.Request) []*collaboration.Recei
 
 	lrs := collaboration.ListReceivedSharesRequest{}
 	// TODO(refs) handle error...
-	shares, _ := c.ListReceivedShares(r.Context(), &lrs)
+	shares, err := c.ListReceivedShares(r.Context(), &lrs)
+	if err != nil {
+		panic(err)
```
Comment (refs) [inline internal/http/services/owncloud/ocs/shares.go]: do we really want to panic?

Comment (labkode) [reply internal/http/services/owncloud/ocs/shares.go]: Not really, just quick workaround. All that code needs propper error handling, you created that first mockup, can you followup on this PR and update it correctly?

Comment (refs) [reply internal/http/services/owncloud/ocs/shares.go]: Will update all that with proper error handling, it's about time I do it :)

Comment (labkode) [reply internal/http/services/owncloud/ocs/shares.go]: @refs any update on this?

**Comment by labkode** [comment]
Closing this, needs proper testing

**Pull request closed without merging**